pkg/security/ebpf/c/dentry_resolver.h

```diff
@@ -173,9 +185,9 @@ int __attribute__((always_inline)) resolve_dentry_tail_call(void *ctx, struct de
     }
 
     if (input->iteration == DR_MAX_TAIL_CALL) {
-        map_value.name[0] = 0;
-        map_value.parent.mount_id = 0;
-        map_value.parent.ino = 0;
+        map_value->name[0] = 0;
+        map_value->parent.mount_id = 0;
+        map_value->parent.ino = 0;
         bpf_map_update_elem(&pathnames, &next_key, &map_value, BPF_ANY);
```

Comment:
shouldn't this be ?
```suggestion
        bpf_map_update_elem(&pathnames, &next_key, map_value, BPF_ANY);
```
also I don't think you can update_elem with a value coming from another maps on 4.14

Reply:
good catch, but why the CI pass ?

Reply:
I don't think it does https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/165040117

Reply:
ah you are in allow to failed ...
yeah map value to map is in 4.18 https://github.com/torvalds/linux/commit/d71962f3e627b5941804036755c844fabfb65ff5

Reply:
so I push the previous solution but it's reducing by 10 chars your path
Is it acceptable ?

Reply:
not really no, let me get back to you on what could work